src/pages/resource/PublicationPanel.tsx

```diff
@@ -16,6 +16,45 @@ interface PublicationPanelProps {
   tabNumber: number;
 }
 
+const StyledPublicationPanel = styled.div`
+  width: 100%;
+  padding-top: 2rem;
+
+  > header {
+    font-size: 2rem;
+    font-weight: bold;
+    line-height: 1.5rem;
+    margin-bottom: 2rem;
+  }
+
+  > section {
+    display: flex;
+    justify-content: space-between;
+  }
+`;
```

Comment:
Hva syns dere om å sette styling for child tags vs å lage flere eksplisitte styled-components?

Reply:
akkurat den der komponenten er generell for alle tab'ene, så jeg synes det burde vært trukket ut i en egen komponent i f.eks `resource/components`. men er åpen for idéen om å ha child tags i en styled component.